x-pack/test/functional_synthetics/apps/uptime/synthetics_integration.ts

```diff
@@ -54,6 +54,7 @@ export default function (providerContext: FtrProviderContext) {
         },
         id: `${getSyntheticsPolicy(agentFullPolicy)?.streams?.[0]?.id}`,
         name,
+        origin: 'ui',
```

Comment:
I am not sure why this value appeared when moving to the v2 registry but I can see that there are other tests which expect the origin field to be set so it seemed acceptable to add.